test/sshd/test.sh

```diff
@@ -5,7 +5,9 @@ set -e
 # Optional: Import test library
 source dev-container-features-test-lib
 
-echo 'NO TESTS FOR ME YET!'
+check "shh-init-exists" bash -c "ls /usr/local/share | grep ssh-init.sh"
```

Comment:
Typos on these check names here and below:
```suggestion
check "ssh-init-exists" bash -c "ls /usr/local/share | grep ssh-init.sh"
```